solarforecastarbiter/reference_forecasts/main.py

```diff
@@ -168,18 +172,23 @@ def run_persistence(observation, forecast, run_time, issue_time, index=False):
     data_start, data_end = get_data_start_end(
         observation, forecast, run_time)
 
+    def load_data():
```

Comment:
doesn't load data take observation, data_start, data_end arguments? This shouldn't work with how it's called in persistence.py. And if those parameters don't matter, why bother passing the load_data function instead of just the data?

Reply:
the test_main functions are not thorough enough if they don't catch this me thinks

Reply:
Ah, yeah that's wrong. Test passed because I mock the calls to `persistence.py` functions. 

I'm willing to revisit the question of passing the data vs. parameters/functions to load the data, but I think we should finish this PR first.